locate/src/config/constants.js

```diff
@@ -45,8 +45,7 @@ const prodConfig = {
     "http://34.78.78.202:30001/api/v1/data/channels/maintenance/add",
   REGISTER_DEVICE_URI: "http://34.78.78.202:30002/api/v1/devices/ts",
   ALL_DEVICES_URI: "http://34.78.78.202:30002/api/v1/devices",
-  DEPLOY_DEVICE_URI:
-    "http://34.78.78.202:30002/api/v1/devices/ts/deploy/device",
+  DEPLOY_DEVICE_URI: "http://34.78.78.202:30002/api/v1/devices/ts/activity?type=",
```

Comment:
Does this URL need to have the query parameter (`type`) on it?
How are we using it inside our REST client library?
@lilly-muyama